src/test/it/cdkata.yml

```diff
@@ -0,0 +1,157 @@
+        source:
+          version: 2.9
+      - groupId: "org.jenkins-ci.plugins.workflow"
+        artifactId: "workflow-support"
+        source:
+          version: 2.19
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "junit"
+        source:
+          version: 1.24
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "mailer"
+        source:
+          version: 1.21
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "scm-api"
+        source:
+          version: 2.2.7
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "script-security"
+        source:
+          version: 1.44
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "aws-java-sdk"
+        source:
+          version: 1.11.341
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "aws-credentials"
+        source:
+          version: 1.23
+        # dependencies to run Pipeline jobs
+      - groupId: "org.jenkins-ci.plugins.workflow"
+        artifactId: "workflow-cps"
+        source:
+          version: 2.54
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "timestamper"
+        source:
+          version: 1.8.10
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "pipeline-utility-steps"
+        source:
+          version: 2.1.0            
+      - groupId: "org.jenkins-ci.plugins.workflow"
+        artifactId: "workflow-durable-task-step"
+        source:
+          version: 2.19 
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "pipeline-stage-step"
+        source:
+          version: 2.3
+      - groupId: "io.jenkins.plugins"
+        artifactId: "aws-global-configuration"
+        source:
+          version: 1.0
+      - groupId: "io.jenkins.plugins"
+        artifactId: "artifact-manager-s3"
+        source:
+          version: 1.1
+          dir: ../../..
+    systemProperties: {
+      java.util.logging.config.file: /var/jenkins_home/log.properties,
+      jenkins.model.Jenkins.logStartupPerformance: true,
+      artifact-manager-s3.enabled: true,
+      io.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.deleteArtifacts: true,
+      io.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.deleteStashes: true,
+      jenkins.install.runSetupWizard: false}
+#    casc:
+#      - id: "initScripts"
+#        source:
+#          dir: casc.yml
+    groovyHooks:
+      - type: "init"
+        id: "initScripts"
+        source:
+          dir: initScripts
+ath:
+  # TODO: ATH test dependency issues
+  disabled: true
+  useLocalSnapshots: false
+  pluginEvaluationOutcome: skipOnInvalid
+  tests:
+    - "core.ArtifactsTest"
+#    - "plugins.CopyArtifactPluginTest"
+#    - "plugins.CompressArtifactsPluginTest"
+#    - "plugins.WorkflowPluginTest"
+#    - "plugins.WorkflowMultibranchTest"
+  categories:
+    - "org.jenkinsci.test.acceptance.junit.SmokeTest"
+pct:
+  disabled: true
+  useLocalSnapshots: false
+  jth:
+    version: 2.38
```

Comment:
Should be omitted. Every plugin repo defines a compatible version of JTH and this should not be overridden.